shared/settings/routes.native.js

```diff
@@ -72,3 +73,30 @@ const routeTree = () => {
 }
 
 export default routeTree
+
+export const newRoutes = {
+  [Constants.aboutTab]: {getScreen: () => require('./about-container').default},
+  [Constants.advancedTab]: {getScreen: () => require('./advanced/container').default},
+  [Constants.chatTab]: {getScreen: () => require('./chat/container').default},
+  [Constants.walletsTab]: {
+    getScreen: () =>
+      isMobile
+        ? require('../wallets/wallet/container').default
+        : require('../wallets/wallets-and-details/container').default,
+  },
+  [Constants.deleteMeTab]: {getScreen: () => require('./delete/container').default},
+  [Constants.feedbackTab]: {getScreen: () => require('./feedback-container').default},
+  [Constants.invitationsTab]: {getScreen: () => require('./invites/container').default},
+  [Constants.landingTab]: {getScreen: () => require('./about-container').default},
+  [Constants.notificationsTab]: {getScreen: () => require('./notifications/container').default},
+  [Constants.passphraseTab]: {getScreen: () => require('./passphrase/container').default},
+  [Constants.screenprotectorTab]: {getScreen: () => require('./screenprotector-container.native').default},
+  dbNukeConfirm: {getScreen: () => require('./db-nuke-confirm/container').default},
+  deleteConfirm: {getScreen: () => require('./delete-confirm/container').default},
+  inviteSent: {getScreen: () => require('./invite-generated/container').default},
+  privacyPolicy: {getScreen: () => require('./web-links.native').default},
+  removeDevice: {getScreen: () => require('../devices/device-revoke/container').default},
+  'tabs:settingsTab': {getScreen: () => require('./').default},
```

Comment:
This is short term cause it's the tabs constants. We can likely change it to something soon. Ultimately all the section routes should be renamed so they at least have a prefix (like settingsAboutPage or whatever and then these could be settingRoot or something)

Reply:
I forked this into `marco/DESKTOP-nav2-with-tabs-renamed` which renamed the tabs to be `tabs.X` so we can use `router.getPathAndParamsFromState` instead of our custom `getVisiblePath`